Document Java Source Context (#7086)
getsentry/sentry-docs: src/wizard/java/index.md

}
```

> To upload your source code to Sentry and show it in stacktraces, use our Gradle plugin.

Reviewer: ```suggestion
To upload your source code to Sentry so it can be shown in stack traces, use our Gradle plugin.
```

</dependency>
```

> To upload your source code to Sentry and show it in stacktraces, use our Maven plugin.

Reviewer: ```suggestion
To upload your source code to Sentry so it can be shown in stack traces, use our Maven plugin.
```

                <!-- minimum required version is 2.17.3 -->
                <sentryCliExecutablePath>/path/to/sentry-cli</sentryCliExecutablePath>

                <org>___ORG_SLUG___</org>

                <project>___PROJECT_SLUG___</project>

                <!-- in case you're self hosting, provide the URL here -->
                <!--<url>http://localhost:8000/</url>-->

                <!-- provide your auth token via SENTRY_AUTH_TOKEN environment variable -->
                <!-- you can find it in Sentry UI: Settings > Account > API > Auth Tokens -->
                <authToken>${env.SENTRY_AUTH_TOKEN}</authToken>
            </configuration>
            <executions>
                <execution>
                    <phase>install</phase>
                    <goals>
                        <goal>uploadSourceBundle</goal>
                    </goals>
                </execution>
            </executions>
        </plugin>
    </plugins>
    ...
</build>
```

### SBT

For **SBT**:

```scala
libraryDependencies += "io.sentry" % "sentry" % "{{@inject packages.version('sentry.java', '4.0.0') }}"
```

> To upload your source code to Sentry and show it in stacktraces, please refer to [Manually Uploading Source Context](https://docs.sentry.io/platforms/java/source-context).

Reviewer: ```suggestion
To upload your source code to Sentry so it can be shown in stack traces, please refer to [Manually Uploading Source Context](https://docs.sentry.io/platforms/java/source-context).
```
